Guide.docc/Guidance.md

```diff
@@ -0,0 +1,69 @@
+# Guidance
```

Comment:
I think the title should be more specific. Most of the guide is guidance on specific topics. Maybe something like "Migration Strategy"?

Reply:
ha yeah